.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,36 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master", branch/* ]
+  pull_request:
+    branches: [ "master", branch/* ]
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'go', 'javascript', 'python' ]
```

Comment:
Do we have any Python code?

Reply:
Technically, we have:
* `./examples/aws/terraform/ha-autoscale-cluster/ansible/ec2.py`
* `./assets/monitoring/convert.py`

But to save build time, I'll remove it.